_sass/components/_accordions.scss

```diff
@@ -22,15 +22,42 @@
       content: "\f077";
     }
 
-    .accordion-content {
+    .accordion-button_more {
+      display: none;
+    }
+
+    .accordion-button_less {
+      display: inline-block;
+
+      &::after {
+        content: '–';
+      }
+    }
+
+    .accordion-content,
+    .accordion-more {
       display: inline-block;
     }
+
   }
 
   &[accordion-open=false] {
     .accordion-button:after {
       content: "\f078";
     }
+
+    .accordion-button_more {
+      &::after {
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets